source/MaterialXRuntime/Private/Commands/PvtConnectionCmd.cpp

```diff
@@ -90,4 +95,33 @@ void PvtConnectionCmd::breakConnection(RtCommandResult& result)
     }
 }
 
+void PvtConnectionCmd::updateConnectionProperties(const RtOutput& /*src*/, const RtInput& /*dest*/, RtCommandResult& /*result*/) 
+{
+    // No-op
+}
+
+
+PvtCommandPtr PvtInterfaceConnectionCmd::create(const RtOutput& src, const RtInput& dest, ConnectionChange change)
+{
+    return std::make_shared<PvtInterfaceConnectionCmd>(src, dest, change);
+}
+
+void PvtInterfaceConnectionCmd::updateConnectionProperties(const RtOutput& src, const RtInput& dest, RtCommandResult& /*result*/)
```

Comment:
Derived class allows for interface properties to be updated as needed. Here we update the token property. It could be we are also missing other properties (metadata) but didn't check too closely.